Tensile/Configs/rocblas_sgemm_example.yaml

```diff
@@ -0,0 +1,80 @@
+GlobalParameters:
+  MinimumRequiredVersion: 4.4.0
+  PrintLevel: 1
+  ForceRedoBenchmarkProblems: True
+  ForceRedoLibraryLogic: True
+  ForceRedoLibraryClient: True
+  CMakeBuildType: Release
+  EnqueuesPerSync: 5
+  SyncsPerBenchmark: 3
+  LibraryPrintDebug: False
+  NumElementsToValidate: 128
+  ValidationMaxToPrint: 4
+  ValidationPrintValids: False
+  ShortNames: False
+  MergeFiles: False
+  Platform: 0
+  Device: 0
+  KernelTime: True
+  PinClocks: False # Must be set to False.
+  SleepPercent: 0
+  DataInitTypeBeta : 0
+  PrintSolutionRejectionReason: True # Route rejected configs to std out.
+  PrintWinnersOnly: 0
+
+BenchmarkProblems:
+  ########################################
+  # NN - standard
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
+      DataType: s
+      TransposeA: False
+      TransposeB: False
+      UseBeta: True
+      Batched: True
+
+    - # BenchmarkProblemSizeGroup - standard
+      InitialSolutionParameters:
+      # Empty for simplicity. 
+      # Please see https://github.com/ROCmSoftwarePlatform/Tensile/wiki/Benchmark-Protocol
+      # for a detailed explanation.
+      BenchmarkCommonParameters:
+      # Specify parameters that are to be considered "determined"
+      # prior to kernel forking.
+        - ProblemSizes:
+          - Exact: [1024, 1024, 1, 512]
```

Comment:
ProblemSizes in anything but BenchmarkFinalParameters no longer do anything

Reply:
Removed.